tests/unittests/core/worker/test_trial_pacemaker.py

```diff
@@ -35,15 +35,17 @@ def trial(exp):
     trial.status = "reserved"
     trial.heartbeat = heartbeat
 
-    get_storage().register_trial(trial)
+    setup_storage().register_trial(trial)
 
     return trial
 
 
-@pytest.mark.usefixtures("storage")
 def test_trial_update_heartbeat(exp, trial):
     """Test that the heartbeat of a trial has been updated."""
-    trial_monitor = TrialPacemaker(trial, wait_time=1)
+
+    storage = setup_storage()
```

Comment:
Maybe it would be best to use orionstate as a fixture instead to avoid polluting local DB.

Reply:
Ah, it's set in `exp` fixture. Alright.